beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -225,13 +341,80 @@ def normalize(tensor):
 plt.imshow(fbank.t().numpy(), cmap='gray')
 
 
+######################################################################
+# You can create mel frequency cepstral coefficients from a raw audio signal
+# This matches the input/output of Kaldi’s compute-mfcc-feats.
+# 
+
+mfcc = torchaudio.compliance.kaldi.mfcc(waveform, **params)
+
+print("Shape of mfcc: {}".format(mfcc.size()))
+
+plt.figure()
+plt.imshow(mfcc.t().numpy(), cmap='gray')
+
+######################################################################
+# Finally, you can resample your waveform at a new frequency, matching Kaldi’s
+# OfflineFeatureTpl ResampleWaveform, which uses a LinearResample.
+# 
+
+NEW_FREQ = sample_rate / 2
+resample_wf = torchaudio.compliance.kaldi.resample_waveform(waveform, sample_rate, NEW_FREQ)
+
+print("Shape of resample_wf: {}".format(resample_wf.size()))
+
+
+######################################################################
+# Available Datasets
+# -----------------
+# 
+# If you do not want to create your own dataset to train your model, ``torchaudio`` offers a
+# unified dataset interface. This interface supports lazy-loading of files to memory, download 
+# and extract functions, and datasets to build models.
+# 
+# The datasets ``torchaudio`` currently supports are:
+#
+# -  **VCTK**: Speech data uttered by 109 native speakers of English with various accents
+#    (`Read more here <https://homepages.inf.ed.ac.uk/jyamagis/page3/page58/page58.html>`_).
+# -  **YESNO**: Sixty recordings of one individual saying yes or no in Hebrew; each
+#    recording is eight words long (`Read more here <https://www.openslr.org/1/>`_).
+# -  **CommonVoice**: An open source, multi-language dataset of voices that anyone can use
+#    to train speech-enabled applications (`Read more here <https://voice.mozilla.org/en/datasets>`_).
+# -  **LibriSpeech**: Large-scale (1000 hours) corpus of read English speech (`Read more here <http://www.openslr.org/12>`_).
+# 
+# To access these datasets, we must import ``common_utils`` to create our asset directory.
+#
+
+import common_utils
+
+######################################################################
+# Next, we create our temporary directory and load our desired dataset.
+#
+
+test_dirpath, test_dir = common_utils.create_temp_assets_dir()
+path = os.path.join(self.path, "commonvoice")
+
+data = torchaudio.datasets.commonvoice.COMMONVOICE(path, "train.tsv", "tatar")
```

Comment:
When the tutorial will be imported in `pytorch.org`, this command will be run, and will download the whole Common Voice dataset. That's too big, so let's not do that. Do you have suggestions on how to illustrate the datasets?

VCTK and YESNO are smaller, so those could be run, and then the other two could be commented out, with a sample of that their output look like for instance.